# Goal

uint8 ACTION_CALIBRATE = 1
uint8 ACTION_DETECT = 2
uint8 action # either ACTION_CALIBRATE or ACTION_DETECT

# Data to use
string topic
bool use_world
bool use_eye0
bool use_eye1 # NOTE: if use_eye0 is false, use_eye1 will not be checked!

# Termination conditions. If none are specified, runs until preempted
time end_time # specify a time to auto-finish collecting data, optional
int32 num_points # specify an amount of data to collect, optional
float64 detection_time # length in seconds to collect data

# TODO: Lots of classification parameters
float64 label_combination_period # sec
float64 min_fix_duration # sec

# For calibration:
string log_dir    # directory to save calibrated model + data to
string load_dir   # directory to load calibration model from (should be output of log_dir)

---
# Result
# all we really need is a bool success and a string message which are implemented by the action server
# so this is empty
# could maybe do info about fixations collected but... why :)

---
# Feedback
# do we need this? could return fixations as they're detected but again... why

